tasks/publish-index-image/README.md

```diff
@@ -11,9 +11,13 @@ Publish a built FBC index image using skopeo
 | targetIndex    | Pullspec to push the image to                                           | No       |               |
 | retries        | Number of skopeo retries                                                | Yes      | 0             |
 | requestTimeout | Max seconds waiting for the status update                               | Yes      | 360           |
+| buildTimestamp | Build timestamp for the publishing image                                | No       |               |
 
 ## Changelog
 
+### Changes in 1.1.0
```

Comment:
It's a breaking change, because it will break without supplying the new parameter. So should be `2.0.0`.